admin/roles/class-role-manager-vip.php

```diff
@@ -7,19 +7,27 @@
 
 /**
  * VIP implementation of the Role Manager.
+ *
+ * @deprecated 19.7
+ * @codeCoverageIgnore
  */
 final class WPSEO_Role_Manager_VIP extends WPSEO_Abstract_Role_Manager {
 
 	/**
 	 * Adds a role to the system.
 	 *
+	 * @deprecated 19.7
+	 * @codeCoverageIgnore
+	 *
 	 * @param string $role         Role to add.
 	 * @param string $display_name Name to display for the role.
 	 * @param array  $capabilities Capabilities to add to the role.
 	 *
 	 * @return void
 	 */
 	protected function add_role( $role, $display_name, array $capabilities = [] ) {
+		_deprecated_function( __METHOD__, 'WPSEO 19.7' );
```

Comment:
Looks like the PR was tagged with the 19.9 milestone, but this line and the one below suggest this method was deprecated in 19.7.

Reply:
I checked :), there was a subsequent commit to increment the deprecation versions.